src/plugins/unified_search/public/filter_bar/filter_editor/lib/filter_editor_utils.ts

```diff
@@ -20,7 +20,7 @@ export function getFieldFromFilter(filter: Filter, indexPattern?: DataView) {
 
 export function getOperatorFromFilter(filter: Filter) {
   return FILTER_OPERATORS.find((operator) => {
-    return filter.meta.type === operator.type && filter.meta.negate === operator.negate;
```

Comment:
@ppisljar, can you tell me, please, why this part of the code has been added and if is it important? It is not working with the changes you requested.

Reply:
it tries to determine the correct operation from the meta properties. I think we should leave the negate part in there then as it seems it can be set in different ways than user negating the filter ? (cc @lukasolson )

Reply:
When creating a filter "is not" (or is not within range, etc.) we set the "negate" property, so this is required.